helpers/notification_helper.py

```diff
@@ -40,6 +49,55 @@ def send_subscription_update(cls, user_id, sending_device_key=""):
         notification.send(clients)
 
     @classmethod
+    def send_upcoming(cls, live_events):
+        from helpers.match_helper import MatchHelper
```

Comment:
Yes, but I can't remember why. I think it caused an import loop somewhere - I'll test it out after I fix everything else